bin/pc2installservice

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+#
+# Purpose:
+#   This script installs a Linux service that runs the script to create
+#   the cgroup necessary for PC2 sandboxes.  The service is a one-shot service
+#   that simply runs the /usr/pc2/bin/pc2installsandbox shell script
+#
+# Author: John Buck
+
+#DEBUG=
+DEBUG="echo Would"
```

Comment:
Shouldn't the above two lines be commented the other way around?  It looks to me like currently the script isn't going to actually do anything...

Reply:
Yeah, I was just putting that there to see if anyone actually reads it. ;-)    No, it's wrong.  Will fix.